baselines/dom.generated.d.ts

```diff
@@ -14392,7 +14396,7 @@ interface SVGSVGElement extends SVGGraphicsElement, DocumentEvent, SVGFitToViewB
     forceRedraw(): void;
     getComputedStyle(elt: Element, pseudoElt?: string | null): CSSStyleDeclaration;
     getCurrentTime(): number;
-    getElementById(elementId: string): Element;
+    getElementById<E extends Element = HTMLElement>(elementId: string): E | null;
```

Comment:
`<svg>` elements can be nested inside SVG trees. `Element` seems like a better default.

Reply:
Please read the entire thread and  #1151 before repeating comments.